features/testbot/matching.feature

```diff
@@ -197,13 +197,50 @@ Feature: Basic Map Matching
             | trace | matchings | geometry                                   |
             | abd   | abd       | 1,1,1.000089,1,1.000089,1,1.000089,0.99991 |
 
-    Scenario: Testbot - Speed greater than speed threshhold, should split -- returns trace as abcd but should be split into ab,cd
-        Given a grid size of 10 meters
+    Scenario: Testbot - Geometry details in polyline
```

Comment:
Hmm, looks like either this branch needs to be rebased on master, or you've accidentally removed some test code.